### PR TITLE
chore: fetch apis in parallel

### PR DESCRIPTION
**Pull request by mcansh**
note: just realized that #36 is a thing, feel free to close this one for that :)

Signed-off-by: Logan McAnsh <logan@mcan.sh>

### CODE REVIEW

File: site/app/routes/index.jsx [hunk 1]
```diff
@@ -8,21 +8,16 @@ import { SectionNextEpisode } from '../components/section-next-episode.jsx';
...
+  let [episodes, schedule, featured, sponsors] = await Promise.all([
+    loadAllEpisodes(),
+    ...[
+      'https://www.learnwithjason.dev/api/schedule',
+      'https://www.learnwithjason.dev/api/episodes/featured',
+      'https://www.learnwithjason.dev/api/sponsors',
+    ].map((url) => fetch(url).then((res) => res.json())),
+  ]);
+
+  return json({ episodes, schedule, featured, sponsors });
```
Comment (ryankshaw) [inline site/app/routes/index.jsx:20]: @mcansh I noticed from your profile picture you work for Remix so I figured I'd ask you: Just wondering, why did you use the `json` helper from remix here in this loader? why not just `return { episodes, schedule, featured, sponsors }`?

Is there some benefit to using the `json` helper here over returning a plain object? Thanks for helping me learn and thank you and all the remix guys for making a great framework!

Comment (mcansh) [reply site/app/routes/index.jsx:20]: there's no _real_ benefit in this scenario as we're not providing any other response info (headers, status code) in either case, Remix will check what's returned and if it's already an instance of a Response, it'll carry on its way, if not, it'll take the result, and put `json` around it https://github.com/remix-run/remix/blob/c746c6f75fb8a74da92df91113df5af7bf87adea/packages/remix-server-runtime/data.ts#L60 - i accidentally left while trying to get Netlify's CDN to play nice #39

Comment (ryankshaw) [reply site/app/routes/index.jsx:20]: cool, that's what I thought too. I just wondered when you put it in there if there was something I was missing.

thanks for the response!

Comment (ryankshaw) [reply site/app/routes/index.jsx:20]: also, did you see my other pull request that does the same thing you are doing here? is there any reason you think this PR would work better than that one?

Comment (mcansh) [reply site/app/routes/index.jsx:20]: I did after i opened this haha, going to close mine as a dupe :)

**Comment by mcansh** [comment]
duplicate of #36

File: site/app/routes/index.jsx (hunk 1 above)
Comment (ryankshaw) [reply site/app/routes/index.jsx:20]: ha ha, sounds good 😄

**Pull request closed without merging**